dashboard.py

```diff
@@ -59,46 +99,45 @@ def model_predictions_review(cfg: DictConfig):
     with cols[3]:
         st.markdown("### Predicted")
 
-    model, dataset = prepare_model_and_dataset_from_checkpoint(checkpoint, cfg)
-
-    n_samples = 5
-    idxs = np.random.randint(len(dataset), size=n_samples)
-
-    records = [dataset.records[idx] for idx in idxs]
-    samples = [dataset[idx] for idx in idxs]
-
-    pad_idx = dataset.tgt_vocab.index("<blank>")
-    bins = train_cfg.dataset.bins.replace(" ", "-")
-
     # predict velocities and get src, tgt and model output
     print("Making predictions ...")
-    for record, sample, idx in zip(records, samples, idxs):
-        result = predict_sample(
-            record=sample,
-            dataset=dataset,
+    for record_id in idxs:
+        # Numpy to int :(
+        record = dataset.get_complete_record(int(record_id))
+        record_source = json.loads(record["source"])
+        src_token_ids = record["source_token_ids"]
```

Comment:
"source_token_ids" are also used here, so you can also change just the train.py file

Reply:
I wanted to have `..._token_ids` everywhere, but I made some typos :(

Reply:
okay!!